env_cylc.py

```diff
@@ -115,33 +110,29 @@ def get_tseries_resubmit(self, ts_tper, ts_n, stop_n, stop_option):
             freq = (ts/cesm)+1
         else:
             freq = (ts/cesm)
-        return freq  
-               
-         
+        return freq
+
+
 
     def get_env(self):
         my_case = os.getcwd() + '/../'
-        
+
         case = Case(my_case, read_only=False)
-        
+
         cwd = os.getcwd()
         os.chdir(my_case)
 
-        env_case = EnvCase()
```

Comment:
All of these get_value arguments are available from case, no need to create objects for each file interface.